apps/dashboard/src/components/subscribers/locale-select.tsx

```diff
@@ -1,71 +1,111 @@
 import { locales } from '@/utils/locales';
-import { RiEarthLine } from 'react-icons/ri';
-import { CountryFlags } from '../icons/country-flags';
-import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '../primitives/select';
+import { cn } from '@/utils/ui';
+import { RiArrowDownSLine, RiCheckLine, RiEarthLine } from 'react-icons/ri';
+import { type Country } from 'react-phone-number-input';
+import flags from 'react-phone-number-input/flags';
+import { Button } from '../primitives/button';
+import { Command, CommandEmpty, CommandGroup, CommandInput, CommandItem, CommandList } from '../primitives/command';
+import { Popover, PopoverContent, PopoverTrigger } from '../primitives/popover';
+import { ScrollArea } from '../primitives/scroll-area';
 import TruncatedText from '../truncated-text';
+import { useState } from 'react';
 
 export function LocaleSelect({
```

Comment:
shouldn't it be possible to clear the locale and timezone? I don't see the option

Reply:
now that you mention it none of our Select fields have clear functionality 😓 🤦🏼

Reply:
Will discuss with Naveen how we will want to introduce a clear functionality

Reply:
Will address this in a different PR as this is something that needs work on the DS level too